tdrs-backend/tdpservice/users/api/utils.py

```diff
@@ -18,45 +18,6 @@
 logger = logging.getLogger()
 logger.setLevel(logging.INFO)
 
-"""
```

Comment:
this looks like a section of code that got merged into `raft-tdp-main` without government review. It would be helpful to have another pr on that.

Reply:
I'll create one for you to take a look at , this was us migrating code out of login.py into a helper class.